### ISSUE TITLE
AxiosError: Request failed with status code 404 when let's encrypt fails

### ISSUE THREAD

**Issue by GlitchWitch** [bakerkretzmar/laravel-deploy-preview#19]
When running this script a 404 error occurs during the clean up step. 

```
Run bakerkretzmar/laravel-deploy-preview@v2
Creating new deploy preview site named 'test'
Site created!
Obtaining SSL certificate
SSL certificate obtained!
Installing 'REDACTEDORG/REDACTEDREPO' Git repository in site
Repository installed!
Updating .env file
Updated .env file!
Setting up scheduler
Scheduled job command set up!
Updating deploy script
Updated deploy script!
Enabling Quick Deploy
Quick Deploy enabled!
Deploying site
Site deployed!
Cleaning up...

file:///home/runner/work/_actions/bakerkretzmar/laravel-deploy-preview/v2/webpack:/laravel-deploy-preview/node_modules/axios/lib/core/settle.js:19
    reject(new AxiosError(
^
AxiosError: Request failed with status code 404
    at settle (file:///home/runner/work/_actions/bakerkretzmar/laravel-deploy-preview/v2/webpack:/laravel-deploy-preview/node_modules/axios/lib/core/settle.js:19:1)
    at Unzip.handleStreamEnd (file:///home/runner/work/_actions/bakerkretzmar/laravel-deploy-preview/v2/webpack:/laravel-deploy-preview/node_modules/axios/lib/adapters/http.js:572:1)
    at Unzip.emit (node:events:526:[35](https://github.com/REDACTEDORG/REDACTEDREPO/actions/runs/6598974758/job/17927500504#step:3:36))
    at endReadableNT (node:internal/streams/readable:1[37](https://github.com/REDACTEDORG/REDACTEDREPO/actions/runs/6598974758/job/17927500504#step:3:38)6:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
```

Interestingly here, it shows `SSL certificate obtained!` despite that step failing due to our environment not being accessible to let's encrypt.

**Comment by GlitchWitch** [comment]
This seems to be caused by let's encrypt failing to obtain a certificate despite the message indicating it succeeded

**Comment by bakerkretzmar** [comment]
I don't think there's actually any way through the Forge API to be sure that obtaining the certificate failed, right now I'm just assuming that if it disappears that's what happened. I can try to handle that specific error more gracefully though.

**Comment by bakerkretzmar** [comment]
Fixed in v2.1.4 (error is handled and we output a detailed message specific to this possible failure).
